src/graph/entities/graph_entity.h

```diff
@@ -28,6 +28,7 @@ typedef GrB_Index NodeID;
 typedef GrB_Index EdgeID;
 
 typedef enum GraphEntityType {
+	GETYPE_UNKNOWN,
```

Comment:
Isn't this dangerous, is that compatible with old RDBs?

Reply:
`GraphEntityType` never gets serialized; prior to now it had only been used in `op_update.c`. I deleted the (nearly identical) enum `EntityType` to declutter a bit.